### ISSUE TITLE
[Metaclub] holy fucking shit this piping!

### ISSUE THREAD

**Issue by Angelite191** [d3athrow/vgstation13#2516]
Just a few things I found blatantly wrong with this station. It's pretty devastating to gameplay, I can't imagine what would have happened to atmos that round if me or Eve-01 weren't playing.

![skeleton fortress 15 2015-01-13 141507](https://cloud.githubusercontent.com/assets/5303834/5727562/0244420a-9b32-11e4-84cd-9342eeca84ef.png)
![skeleton fortress 15 2015-01-13 140045](https://cloud.githubusercontent.com/assets/5303834/5727561/0243e710-9b32-11e4-80ed-50a219f2e763.png)
![skeleton fortress 15 2015-01-13 134236](https://cloud.githubusercontent.com/assets/5303834/5727560/02432bb8-9b32-11e4-8e0a-e0f7adc8ed67.png)
![skeleton fortress 15 2015-01-13 134001](https://cloud.githubusercontent.com/assets/5303834/5727564/024516f8-9b32-11e4-929d-e213fc9b79db.png)
![skeleton fortress 15 2015-01-13 133556](https://cloud.githubusercontent.com/assets/5303834/5727563/0244c0a4-9b32-11e4-926f-d12fc6576025.png)

Also, we have a super matter room, but no super matter shard... why?

**Comment by Probe1** [comment]
by the way you should never put an air vent in the middle of a god damn solar panel room.  people run wires in those rooms.  people. run. wires. in. those. rooms.  putting a vent right there is a nuisance.

**Comment by Duny-** [comment]
Hahaha that last one holy shit I love it

**Comment by IratePirate** [comment]
Bottom two are fixed in the PR I did, just trying to get the power sorted and dealing with exams.

I'll deal with the third one.

2nd one was put there by the original map creator, there were like 8 of those around maint but I got rid of all of the others. Forgot to get rid of that one. It was a hairbrained idea to put them in in the first place.

Top one was a mistake by the original creator i'd assume, i'll delete it. No solars are supposed to have them.

Thanks for the feedback.
